CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing
```

Comment:
this can also go in `.github/`

Reply:
yes, i've removed the duplicate, updated the Readme to refer to the existing file in the .github/ directory, and updated the file